upgrade_test.py

```diff
@@ -1438,6 +1442,38 @@ def _update_argus_upgraded_version(self, node, new_version):
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
 
+    def validate_limited_voters_feature_enabled(self):
+        InfoEvent(message="Check that limited voters feature enabled after upgrade for all nodes")
+        max_voter_number = 5
+        feature_state_per_node = []
+        for node in self.db_cluster.nodes:
+            result = wait_for(func=self.db_cluster.is_features_enabled_on_node,
+                              timeout=60,
+                              step=1,
+                              text=f"Check feature enabled on node {node.name}",
+                              throw_exc=False,
+                              feature_list=["GROUP0_LIMITED_VOTERS"],
+                              node=node)
+            feature_state_per_node.append(result)
+        if all(not enabled for enabled in feature_state_per_node):
```

Comment:
I think it should be 3 cases here 
1 - **Not enabled** on all nodes - return 
2 - **enabled** on all nodes - go further  
2 - **enabled** but **not on all** nodes - Exception

Reply:
added exception